guide/installation/gmc_install/gmc_install.md

```diff
@@ -0,0 +1,118 @@
+
+
+
+## Install GMC
+
+**Prerequisites**
+
+- A running Kubernetes cluster with at least 16 CPUs, 32GB of memory, and 100GB of disk space. To install an Kubernetes cluster as your platform:
+[Kubernetes installation](../k8s_install/) 
+
+**Build and push your image to the location specified by `CTR_IMG`:**
+
+```sh
+make docker.build docker.push CTR_IMG=<some-registry>/gmcmanager:tag
+```
+
+**NOTE:** This image ought to be published in the personal registry you specified.
+And it is required to have access to pull the image from the working environment.
+Make sure you have the proper permission to the registry if the above commands don’t work.
+
+**Install GMC CRD**
+
+```sh
+kubectl apply -f config/crd/bases/gmc.opea.io_gmconnectors.yaml
+```
+
+**Get related manifests for GenAI Components**
+
+```sh
+mkdir -p $(pwd)/config/manifests
+cp $(dirname $(pwd))/manifests/ChatQnA/*.yaml -p $(pwd)/config/manifests/
+cp $(dirname $(pwd))/manifests/DocSum/xeon/docsum_llm.yaml $(pwd)/config/manifests/
+cp $(dirname $(pwd))/manifests/DocSum/gaudi/docsum_gaudi_llm.yaml $(pwd)/config/manifests/
+```
+
+**Copy GMC router manifest**
+
+```sh
+cp $(pwd)/config/gmcrouter/gmc-router.yaml -p $(pwd)/config/manifests/
+```
+
+**Create Namespace for gmcmanager deployment**
+
+```sh
+export SYSTEM_NAMESPACE=system
+kubectl create namespace $SYSTEM_NAMESPACE
+```
+
+**NOTE:** The `SYSTEM_NAMESPACE` should keep the same with the namespace defined in gmc-manager.yaml and gmc-manager-rbac.yaml.
+
+**Create ConfigMap for GMC to hold GenAI Components and GMC Router manifests**
+
+```sh
+kubectl create configmap gmcyaml -n $SYSTEM_NAMESPACE --from-file $(pwd)/config/manifests
+```
+
+**NOTE:** The configmap name `gmcyaml' is defined in gmcmanager deployment Spec. Please modify accordingly if you want
+use a different name for the configmap.
+
+**Install GMC manager**
+
+```sh
+kubectl apply -f $(pwd)/config/rbac/gmc-manager-rbac.yaml
+kubectl apply -f $(pwd)/config/manager/gmc-manager.yaml
+```
+
+**Check the installation result**
+
+```sh
+kubectl get pods -n system
+NAME                              READY   STATUS    RESTARTS   AGE
+gmc-controller-78f9c748cb-ltcdv   1/1     Running   0          3m
+```
+
+## Use GMC to compose a chatQnA Pipeline
+A sample for chatQnA can be found at config/samples/chatQnA_xeon.yaml
+
+**Deploy chatQnA GMC custom resource**
+
+```sh
+kubectl create ns chatqa
+kubectl apply -f $(pwd)/config/samples/chatQnA_xeon.yaml
```

Comment:
use the github url location from genAIexamples. that may make it safer as opposed to relative locations. We do not even expct folks to have done a git cloen then.
If you think this is a bad suggestion, just comment the same and skip it.

Reply:
It's a very good suggestion to use url location instead of cloning github and using the relative location. However, we still have some steps that require users to clone the repo, such as building images. We can change to url location once we have the public images, what do you think?

Reply:
Sure, I will file an issue later.

Reply:
https://github.com/opea-project/docs/issues/39